src/SourceBuild/content/README.md

```diff
@@ -71,45 +71,63 @@ For the latest information about Source-Build support, please watch for announce
 
 ### Prerequisites
 
-The dependencies for building .NET from source can be found [here](https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/linux-requirements.md).
+The dependencies for building can be found [here](https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/).
 In case you don't want to / cannot prepare your environment per the requirements, consider [using Docker](#building-using-docker).
 
 ### Building (source-build configuration)
 
-1. **Clone the VMR**
```

Comment:
This title of this section and the following should no longer include `(source-build configuration)`